src/main/java/frc/robot/subsystems/grabber/Grabber.java

```diff
@@ -36,32 +36,35 @@ public boolean getGamePieceNotDetected() {
     }
 
     public Command stop() {
-        return runOnce(() -> {
-                    targetVoltage = 0.0;
-                    io.stop();
-                }
-        );
+        return runOnce(stopRun());
+    }
+
+    private Runnable stopRun() {
+        return () -> {
+            targetVoltage = 0.0;
+            io.stop();
+        };
     }
```

Comment:
Instead of making a method that returns a `Runnable`, you should it return `void` and use it by `this::stopRun`. In addition, I would rename, as the "run" suffix isn't super descriptive. Because it's an internal method, you could name it `stopInternal` and add a documentation comment saying that this is to help support command implementations.